backend/prisma/schema.prisma

```diff
@@ -66,9 +66,11 @@ model UserListing {
 }
 
 model SpaceListing {
-  id          String  @id @default(cuid())
+  id          String   @id @default(cuid())
```

Comment:
i'm not sure what the change is here, since createdAt and updatedAt fields exist in the main branch too. if it's just whitespace or something then we can safely ignore.